front/app/containers/MainHeader/Components/TenantLogo/index.tsx

```diff
@@ -11,13 +11,14 @@ import messages from '../../messages';
 import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
 
 // components
-import { BoxProps, Image } from '@citizenlab/cl2-component-library';
+import { Image } from '@citizenlab/cl2-component-library';
 
 const LogoLink = styled(Link)`
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  flex: '1 1 auto';
```

Comment:
So was this needed in the end?

Reply:
Not the extra prop, no :)

Reply:
But this flex style is needed? Just that I didn't see a difference yesterday, but I don't know what you're seeing (and what's causing you to introduce this line). :)

Reply:
Yes it's needed - I didn't introduce it but I did end up changing which line it was on (+ I moved all these components, so it looks like I added it):

Original on master:
```
const LogoLink = styled(Link)`
  flex: 1 1 auto;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
`;
```